bin/setenv-docker-customize.sh

```diff
@@ -685,7 +707,7 @@ case "${EXO_DB_TYPE}" in
 esac
 
 # Wait for mongodb availability (if chat is installed)
-if [ -f /opt/exo/addons/statuses/exo-chat.status ]; then
+if [ -f /opt/exo/addons/statuses/exo-chat.status ] && [ "${EXO_CHAT_SERVER_STANDALONE}" = "false" ]; then
```

Comment:
I don't think we need to test both variables

Reply:
yes check remove, it's a remaining of when the client only addon was not available.